### ISSUE TITLE
A question about finetune autoencoder？

### ISSUE THREAD

**Issue by wyhlovecpp** [lewandofskee/DiAD#31]
Hi,
When I finetune the ae on my own dataset, I found that the total number of steps would be divided by 8 because of the config setting [here](https://github.com/lewandofskee/DiAD/blob/ff5f6aaa5413e9ffe56a9cc6892763a00170125f/models/autoencoder_kl_32x32x4.yaml#L49C1-L49C22).

I want to know how many steps you used to finetune, because I found that it took 5w steps to add another loss LPIPSWithDiscriminator in this config.

**Comment by lewandofskee** [comment]
We do not employ the LPIPSWithDiscriminator loss function, as in our experiments, satisfactory reconstruction results were typically achieved within 25,000 steps. Alternatively, one can assess the need for increasing the training steps by examining the visual images in the image_log/train folder.
